esrally/rally.py

```diff
@@ -24,7 +24,7 @@
 import uuid
 
 from esrally import version, actor, config, paths, racecontrol, reporter, metrics, track, chart_generator, exceptions, log
-from esrally import PROGRAM_NAME, DOC_LINK, BANNER, SKULL, check_python_version
+from esrally import PROGRAM_NAME, BANNER, SKULL, doc_link, check_python_version
```

Comment:
nit: this is just me, normally I try to have these in alphabetical order (easier said than done).

Reply:
I tried to keep constants and functions separately that's why I moved the `doc_link`